lib/puppet/pops/types/puppet_object.rb

```diff
@@ -11,6 +11,16 @@ module PuppetObject
   def _ptype
```

Comment:
If the other `_p...` names change to `_pcore_...` then this should also change. i.e. `_pcore_type` instead of `_ptype`.
I know this is more painful - up till now the Pcore Objects have not been exposed anywhere where it is considered API, but now it does.